package.json

```diff
@@ -38,21 +38,21 @@
     "react": "16.x.x"
   },
   "dependencies": {
-    "@stoplight/graph": "file:.yalc/@stoplight/graph",
-    "@stoplight/memoize-one": "4.0.x",
-    "@stoplight/ui-kit": "1.7.x",
-    "react-select": "2.1.x",
-    "ts-essentials": "1.0.x"
+    "@stoplight/memoize-one": "4.0.x"
   },
   "devDependencies": {
-    "@stoplight/scripts": "3.x.x",
+    "@fortawesome/free-solid-svg-icons": "^5.6.1",
+    "@stoplight/scripts": "4.0.0",
+    "@stoplight/storybook-config": "1.x.x",
+    "@stoplight/ui-kit": "1.24.0",
```

Comment:
Shouldn't `@stoplight/ui-kit` be a peer dep?

Reply:
Only because we're using it here: https://github.com/stoplightio/formtron/pull/4/files#diff-68e855f8ad41515fea3926c7d0aa0699R8

Reply:
Didn't I just make it a peer dep?

Reply:
aw fuck I made it a dev dep

Reply:
OK so making it a peer dep makes it fail in Circle CI. What's our strategy? Manually install it in the Circle CI script?

Reply:
oh hang on I bet i forgot to update the lockfile

Reply:
nvm I'm not sure what I'm doing wrong

Reply:
nvm I figured it out